### DISCUSSION TITLE
Add a shared folder

### DISCUSSION THREAD

**Discussion by emanueleg** [ltsp/ltsp#234] [Q&A]
Hello, I'm testing ltsp19: the setup was smooth and everything is working nicely. Congrats to all developers!
I have a working chrootless and now I'd need to activate a shared folder (let's say /home/Shared/) accessible to all students: my use case is exactly the one described in ltsp/ltsp#187 

How is it possible to make this /home/Shared/ (or whatever) directory visible to all clients using the command line (and ltsp19)?

Thanks!

**Comment by enaut** [comment]
The new version of ltsp-manager is not yet able to do this. If you know greek you can use the sch-scripts. if you don't you can manually add a network path to the fstab of each client with ltsp.conf:

In the client section of ltsp.conf:
> FSTAB_x="server:/home /home nfs defaults,nolock,rsize=32768,wsize=32768 0 0"
>    All parameters that start with FSTAB_ are sorted and then their values are written to /etc/fstab at the client init phase.

You need to make the server directory with correct access rights and share it with nfs manually.
source: https://ltsp.org/man/ltsp.conf/
